src/output-json.c

```diff
@@ -398,6 +398,13 @@ void EveAddMetadata(const Packet *p, const Flow *f, JsonBuilder *js)
 void EveAddCommonOptions(const OutputJsonCommonSettings *cfg, const Packet *p, const Flow *f,
         JsonBuilder *js, enum OutputJsonLogDirection dir)
 {
+    if (cfg->include_suricata_version) {
+#ifdef REVISION
+        jb_set_string(js, "suricata_version", PROG_VER " (" xstr(REVISION) ")");
+#else
+        jb_set_string(js, "suricata_version", PROG_VER);
+#endif
```

Comment:
On the event consumer side of things, it would be better if this always had the same format..

Reply:
So what would be the best solution, in this case? Not have the revision, or what could we put in there if there isn't one?

Reply:
Would you want two fields ? `suricata_version` and `suricata_revision` ?

But even then, I have no guarantee that `PROG_VER` will always have the same format... Or what do you mean by `format` here ?

Reply:
I think I like the two fields solution.

Reply:
I might want to group alerts by  Suricata version. I don't want "8.0.0" results, and "8.0.0 <hash>" in my result set because one was built from a git tag, and the other was built from a release package. That is all.

IMO I don't see much need for the revision at all, but can be indifferent about it.

We could also throw a regular expression in the schema to catch a case where the format does change.

Reply:
Went for dropping REVISION